src/Uno.UI/Controls/UnoNavigationBar.iOS.cs

```diff
@@ -2,14 +2,51 @@
 using System.Collections.Generic;
 using System.Text;
 using CoreGraphics;
+using Windows.UI.Xaml;
+
+#if XAMARIN_IOS_UNIFIED
+using Foundation;
 using UIKit;
+#elif XAMARIN_IOS
+using MonoTouch.Foundation;
+using MonoTouch.UIKit;
+#endif
 
 namespace Uno.UI.Controls
 {
-	public partial class UnoNavigationBar : UINavigationBar
+	public partial class UnoNavigationBar : UINavigationBar, DependencyObject
```

Comment:
There should be no need to add `DependencyObject` here.

Reply:
@jeromelaban I don't see InitializeBinder available without deriving from DependencyObject. Should we just tear out the use of generated code here?

Reply:
Indeed, I though it would have been added by another part of the build generation. It can be added then. Thanks :)